app/models/mixins.py

```diff
@@ -0,0 +1,76 @@
+import base64
+import binascii
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.backends import default_backend
+from flask import abort
+import hashlib
+import math
+
+
+class DataObscuringMixin:
```

Comment:
Maybe rename see https://github.com/alphagov/digitalmarketplace-credentials/pull/41/files#r149741874

Reply:
IdentifierObscuringMixin? IDObscuringMixing?

Reply:
Well strictly what you've created *here* isn't id specific - could be used with any integer right?

Reply:
Yep, technically true.